src/icons/service-azure-functions.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
There are multiple colors in this one, so can't do it.